projects/bp-gallery/src/views/shared/picture-info/DescriptionsEditField.tsx

```diff
@@ -90,18 +112,16 @@ const DescriptionsEditField = ({
           </div>
         );
       })}
-      {role >= AuthRole.CURATOR && (
+      {role >= AuthRole.CURATOR && !descriptionState.some(d => isEmpty(d.id)) && (
         <IconButton
           onClick={() => {
-            setDescriptionState(allDescriptions => {
-              return [
-                ...allDescriptions,
-                {
-                  text: '',
-                  id: `${allDescriptions.length + 1}`,
-                },
-              ];
-            });
+            setDescriptionState(allDescriptions => [
+              ...allDescriptions,
+              {
+                text: '',
+                id: ``,
```

Comment:
I don't remember it clearly, but do we send this id to the API at any point? Not really, if we filter all descriptions out with an empty text, right?
I think setting the id to `null` or `undefined` would make it more clear that its a "temporary" id or what do you think? 😅

Reply:
Sadly, our current approach using Strapi does not accept `null` or `undefined` as ids, it seems.

Concernign your other question: We send this to the API as soon as the blur event on the corresponding input field is triggered, thus reducing the probability of `text` being empty. This was done so that we only have to connect to the API at one point (except for deletion): Whenever a text was changed.

Reply:
Okay good point! 👍